sdk/cosmosdb/cosmos/src/ClientContext.ts

```diff
@@ -152,6 +152,8 @@ export class ClientContext {
         diagnosticNode,
         request.resourceType,
         request.operationType,
+        0,
```

Comment:
@amanrao23 @ajnovice  Let's use an iterator instead of index. 
Not for this PR, but maybe in next verison lets plan this.